src/components/calculator.js

```diff
@@ -14,41 +15,47 @@ function Calculator() {
   };
 
   return (
-    <div className="calulator-container">
-      <Result answer={state} />
-      <div className="wrapper-operation">
-        <div>
-          <button onClick={btnClickHandler} type="button" className="other-operations btn">AC</button>
-          <button onClick={btnClickHandler} type="button" className="other-operations btn">+/-</button>
-          <button onClick={btnClickHandler} type="button" className="other-operations btn">%</button>
-          <button onClick={btnClickHandler} type="button" className="other-operations btn operator">&#247;</button>
-        </div>
+    <>
+      <div>
+        <h1 className="header"> Lets Do Some Math! </h1>
+      </div>
+      <div className="calculator-container">
+        <Result answer={state} />
+        <div className="wrapper-operation">
+          <div>
+            <button onClick={btnClickHandler} type="button" className="other-operations btn">AC</button>
+            <button onClick={btnClickHandler} type="button" className="other-operations btn">+/-</button>
+            <button onClick={btnClickHandler} type="button" className="other-operations btn">%</button>
+            <button onClick={btnClickHandler} type="button" className="other-operations btn operator">&#247;</button>
+          </div>
```

Comment:
- [ ] One of the requirements for this project is to make the pages look like the wireframes provided for this project. Kindly make the Calculator Page look identical to the wireframe below 👍 😎 :

![image](https://github.com/riskmaris/math-magicians/assets/77106962/c6306ac6-da5f-4902-85c2-3ea2d579b556)

### Your Design

![image](https://github.com/riskmaris/math-magicians/assets/77106962/6eea6c88-cff8-430a-ab33-0fba4dc85c02)

#### Designing a page according to the provided wireframe ensures consistent user experience and visual alignment with the project's intended functionality. It helps bridge the gap between design and development, minimizing misunderstandings and enhancing the efficiency of the development process. 💻 🖱️